editor/store/actions.js

```diff
@@ -440,7 +440,7 @@ export function createUndoLevel() {
 export function removeBlocks( uids, selectPrevious = true ) {
 	return {
 		type: 'REMOVE_BLOCKS',
-		uids,
+		uids: castArray( uids ),
```

Comment:
Should we update the docs to:
* @param {string|string[]} uids           Block UIDs.